alpine/Dockerfile

```diff
@@ -12,23 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3.6-alpine3.8
+FROM python:3.6-alpine3.10
 
 ENV PYTHONUNBUFFERED=1
 
-RUN apk add --update --no-cache openssl ca-certificates
-
+# Work in the /opt directory to persist the pip-compiled requirement list for
+# child images to use.
 WORKDIR /opt
 
-COPY wsgi-requirements.in ./
+RUN apk update
```

Comment:
I'm not sure about the interaction of this and the `--no-cache` flag later on. Would the flag force another update step each?

Reply:
Also, keeping the list files around is not necessary which is why one typically rolls update, install, and clean up into one Docker layer.

Reply:
Ah, indeed `--no-cache` forces updates every step, so that's a very unhelpful addition. I'll remove it.